rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -1538,6 +1551,40 @@ def test_participant_summary_returns_latest_pm(self):
     self.assertEqual(participant_summary['physicalMeasurementsCreatedSite'],
                      'hpo-site-bannerphoenix')
 
+  def test_switch_to_test_account(self):
+    SqlTestBase.setup_codes(["PIIState_VA", "male_sex", "male", "straight", "email_code", "en",
+                             "highschool", "lotsofmoney"], code_type=CodeType.ANSWER)
+
```

Comment:
New test case for two things:
1. test switch participant to test account
2. make sure `something_changed ` flag in questionnaire_response_dao.py will not be overwrite

Reply:
perhpas add a test case in participant_api_test as well.

Reply:
I'm not sure what to test in participant_api_test, any suggestion?

Reply:
hpo id, version, last_modified.

Reply:
Done